lib/link_expansion/hashed_edition.rb

```diff
@@ -0,0 +1,37 @@
+module LinkExpansion::HashedEdition
+  def edition_hash(values)
+    return nil unless values.present?
```

Comment:
I believe it happens within a test here: 
https://github.com/alphagov/publishing-api/blob/master/spec/lib/link_expansion/content_cache_spec.rb#L19
Not sure if the `ContentCache#find` method could be called with a non-existent `content_id`...
https://github.com/alphagov/publishing-api/pull/976/files#diff-d21d7da8d7d146227416a146fde15973R33